assets/build/print.css

```
@@ -0,0 +1,181 @@
+  padding: 25px;
+  height: auto;
+  background: #F9F9F9;
+  display: block;
+}
+.entry-content pre {
+  font-family: Monaco, Menlo, Consolas, "Courier New", monospace;
+  display: block;
+  padding: 8.5px;
+  margin: 0 0 9px;
+  word-break: break-all;
+  word-wrap: break-word;
```

Stylelint (17.5.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)